powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -162,19 +162,35 @@ void setUp() throws IOException {
 
     @Test
     void testSuccessfulIdentityVerification() throws Exception {
-        String[] context = prepareActivation();
-        String activationId = context[0];
-        String processId = context[1];
+        final TestContext context = prepareActivation();
```

Comment:
I have created a new context object. Passing more parameters of the same type is error prone.